spk/octoprint/src/requirements.txt

```diff
@@ -14,14 +14,14 @@ itsdangerous==0.24
 Jinja2==2.8
 Markdown==2.6.2
 netaddr==0.7.17
-netifaces==0.10.4
+cross:netifaces==0.10.4
```

Comment:
I already have this on my branch for #4096, but since I rebased with master to get requirement.txt and requirements-cross.txt it fails to install with the cross compiled netifaces.
And I don't know what's wrong with it.

Reply:
@hgy59 when did you rebase? In https://github.com/SynoCommunity/spksrc/pull/4920#pullrequestreview-798644475 there is a fix that worked for sickchill.

Reply:
reinstall after uninstall solved the issue with netifaces.